packages/core/content-manager/server/src/history/services/history.ts

```diff
@@ -177,25 +265,130 @@ const createHistoryService = ({ strapi }: { strapi: Core.Strapi }) => {
         getLocaleDictionary(),
       ]);
 
-      const versionsWithMeta = results.map((version) => {
-        return {
-          ...version,
-          meta: {
-            unknownAttributes: getSchemaAttributesDiff(
-              version.schema,
-              strapi.getModel(params.contentType).attributes
-            ),
+      type EntryToPopulate =
+        | {
+            documentId: string;
+            locale: string | null;
+          }
+        | { id: Data.ID }
+        | null;
+
+      /**
+       * Get an object with two keys:
+       * - results: an array with the current values of the relations
+       * - meta: an object with the count of missing relations
+       */
+      const buildRelationReponse = async (
+        values: EntryToPopulate[],
+        attributeSchema: Struct.SchemaAttributes[keyof Struct.SchemaAttributes]
```

Comment:
Is it not the same as `Schema.Attribute.AnyAttribute`?

Reply:
i couldn't find the proper import, thanks!